src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -125,14 +127,18 @@
         return personalisation;
     }
 
-    public void setHearingContactDate(Map<String, String> personalisation, SscsCaseDataWrapper wrapper) {
-        Optional<ZonedDateTime> hearingContactDate = hearingContactDateExtractor.extract(wrapper);
-        if (hearingContactDate.isPresent()) {
-            personalisation.put(
-                AppConstants.HEARING_CONTACT_DATE,
-                formatLocalDate(hearingContactDate.get().toLocalDate())
-            );
+    private String getPanelCompositionByBenefitType(Benefit benefit) {
+        if ("PIP".equals(benefit.name())) {
+            return "judge, doctor and disability expert";
```

Comment:
Do you think this might be better to be set in some sort of config so that any custom email text can be easily found?